server/src/main/java/com/objectcomputing/checkins/services/agenda_item/AgendaItem.java

```diff
@@ -0,0 +1,112 @@
+package com.objectcomputing.checkins.services.agenda_item;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "agenda_items")
+public class AgendaItem {
+
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    private UUID id;
+
+    @NotNull
+    @Column(name = "checkinid")
+    @TypeDef(type = DataType.STRING)
+    private UUID checkinid;
+
+    @NotNull
+    @Column(name = "createdbyid")
+    @TypeDef(type = DataType.STRING)
+    private UUID createdbyid;
+
+    @Nullable
+    @Column(name = "description")
+    private String description;
+
+    public AgendaItem(@JsonProperty("checkinid") @TypeDef(type = DataType.STRING) @Valid @NotNull UUID checkinid,
```

Comment:
we need to remove @JsonProperty annotations and not the @schema ones. @Schema annotations are needed for open Api specifications.

Reply:
That is correct, while swagger will automagically use JavaDoc comments as the schema, its nice to explicitly declare it with the Schema annotation, especially since it gives you control over whether that field is required or not.

Reply:
Can you remove JsonProperty TypeDef Valid and NotNull on the constructor below. That is what I was referring to originally that wasn't needed.

Reply:
Got it, updated the constructor. Thanks.